django-testbed/frontend/src/models/User.js

```diff
@@ -1,19 +1,27 @@
 /* eslint-disable no-use-before-define,@typescript-eslint/no-use-before-define */
 import { Endpoint, PaginatedEndpoint } from '@prestojs/rest';
+import { usePaginator } from '@prestojs/util';
 import {
+    AsyncChoices,
     CharField,
     FilterSet,
     ImageField,
     IntegerField,
     NullableBooleanField,
     NumberField,
+    useViewModelCache,
 } from '@prestojs/viewmodel';
 
 import namedUrls from '../namedUrls';
 
 import BaseUser from './generated/BaseUser';
 
 function transformAndCacheUser(data) {
+    if (data.results && data.count != null) {
+        throw new Error(
+            'Response is paginated but paginator was not defined. Do you need to use PaginatedEndpoint instead?'
+        );
+    }
```

Comment:
This is obsoleted by #68 which removes this check (because it's dependant on how pagination is implemented)

I couldn't think how to do the check generically so left it out but may be worth revisiting. Possible a catch all on errors that says something like 'There was a problem transforming response data. Is the response paginated?' if it fails on an unpaginated Endpoint